src/components/Projects/ProjectDetails.astro

```diff
@@ -12,6 +12,7 @@ type Props = {
   Content: AstroComponentFactory
 };
 const { fields, icon, categoriesData, filters, Content } = Astro.props;
+console.log()
```

Comment:
Did you mean to leave that in there? It doesn't seem to be doing much?

Reply:
done